easybuild/easyconfigs/l/libfabric/libfabric-1.13.0-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,63 @@
+easyblock = 'ConfigureMake'
+
+name = 'libfabric'
+version = '1.13.0'
+
+homepage = 'https://ofiwg.github.io/libfabric/'
+description = """
+Libfabric is a core component of OFI. It is the library that defines and exports
+the user-space API of OFI, and is typically the only software that applications
+deal with directly. It works in conjunction with provider libraries, which are
+often integrated directly into libfabric.
+"""
+
+# The psm3 provider (enabled by default) requires an AVX capable system to run
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+github_account = 'ofiwg'
+source_urls = ['https://github.com/ofiwg/%(name)s/releases/download/v%(version)s']
+sources = [SOURCE_TAR_BZ2]
+patches = [
+    '%(name)s-1.12.1_disable-psm3-avx-config.patch',
+]
+checksums = [
+    '0c68264ae18de5c31857724c754023351614330bd61a50b40cef2b5e8f63ab28',  # libfabric-1.13.0.tar.bz2
+    # libfabric-1.12.1_disable-psm3-avx-config.patch
+    '129be05a824b41eeb7bddac3c23aa421a13f7dde622948df942a265ccc96125a',
+]
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('pkg-config', '0.29.2'),
+    ('Autotools', '20210726'),
+]
+
+dependencies = [
+    ('numactl', '2.0.14'),
+]
+
+osdependencies = [OS_PKG_IBVERBS_DEV]
+
+# Regenerate build files
+preconfigopts = "autoreconf -f -i &&"
```

Comment:
```suggestion
# Add a missing VERSION file in libfabric 1.13.0
preconfigopts = 'echo "3_1_0_0" > prov/psm3/VERSION && '

# Regenerate build files
preconfigopts += 'autoreconf -f -i &&'
```

The PSM3 VERSION file is missing in `libfabric-1.13.0.tar.bz2` and this causes issues when we regenerate the build files due to #13215.

Reply:
Though we should patch that change in - so that it fails if that file is in future releases.

Reply:
I made the patch